File: src/app/objetos/Revista.ts
export class Revista{
    _codigo!:number;
    _nombre:String;

    _precio:number;

    constructor(nombre: String, precio:number){
    this._nombre=nombre;
    this._precio= precio;

    }

    set nombre(nombre:String){
        this.nombre = nombre;
    }

    get nombre(){
        return this.nombre;
    }

    set precio(precio:number){
        this._precio = precio;
    }

    get precio(){
        return this._precio;
    }
    set codigo(codigo:number){
        this._codigo = codigo;
    }

    get codigo(){
        return this._codigo;
    }
}

File: backendRevistas/src/main/java/Modelo/Revista.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package Modelo;

import java.time.LocalDate;

/**
 *
 * @author ordson
 */
public class Revista {
    int codigo;
    String nombre;
    LocalDate fecha = LocalDate.now();
    double precio;

    public Revista(int codigo, String nombre, LocalDate fecha, double precio) {
        this.codigo = codigo;
        this.nombre = nombre;
        this.fecha = fecha;
        this.precio = precio;
    }

    
    public Revista(String nombre, LocalDate fecha, double precio) {
        this.nombre = nombre;
        this.fecha = fecha;
        this.precio = precio;
    }

    public Revista() {
    }

    public int getCodigo() {
        return codigo;
    }

    public void setCodigo(int codigo) {
        this.codigo = codigo;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public LocalDate getFecha() {
        return fecha;
    }

    public void setFecha(LocalDate fecha) {
        this.fecha = fecha;
    }

    public double getPrecio() {
        return precio;
    }

    public void setPrecio(double precio) {
        this.precio = precio;
    }
    
    @Override
    public String toString(){
        return codigo+" "+nombre+"  "+precio+"  "+ fecha;
    }
}
